src/main/webapp/test/mentors.test.js

```diff
@@ -0,0 +1,67 @@
+import {getMentors, createMentorElement, createEmailLink, createLocationHeader, createImg, createBioParagraph, createNameHeader, createGitHubLink, createInterestTagsDiv} from '../mentors.js';
+import '@testing-library/jest-dom/extend-expect';
+
+global.mailtouiApp = {run: () => {}};
+
+const mockMentor = {
+  interestTags: ['interest1', 'interest2'],
+  projects: [{respositoryID: 'mockRepoID', name: 'mockProjectName'}],
+  bio: 'This is the mock bio.',
+  email: 'samialves@google.com',
+  gitHubID: 'mockGitID',
+  image: '../images/dior.jpg',
+  location: 'Mockville, Mocktana',
+  name: 'Mocky the Mockster',
+};
+
+describe('Mentor card', () => {
+  it('is created', () => {
+    const mentorElement = createMentorElement(mockMentor);
+    expect(mentorElement).not.toBeNull();
+  });
+
+  it('has the correct image element', () => {
+    const imgElement = createImg(mockMentor.image);
+    expect(imgElement.getAttribute('class')).toBe('mentor-picture');
+    expect(imgElement.getAttribute('src')).toBe(mockMentor.image);
+  });
+
+  it('has the correct name element', () => {
+    const nameElement = createNameHeader(mockMentor.name);
+    expect(nameElement.getAttribute('class')).toBe('card-title text-primary');
```

Comment:
I wonder if this and similar class checks are necessary for stylistic classes like `text-primary`? Consider using [`toHaveClass`](https://github.com/testing-library/jest-dom#tohaveclass) instead to test for specific required classes that would break functionality if they weren't present.

Reply:
I am really interested in Obi's suggestion of fixtures and in your suggestion of only testing for the presence of certain classes. I will look into both options.